sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -267,50 +261,26 @@ func (m *networkFlowManager) Capabilities() []centralsensor.SensorCapability {
 	return nil
 }
 
-func (m *networkFlowManager) Notify(e common.SensorComponentEvent) {
-	log.Info(common.LogSensorComponentEvent(e))
-	switch e {
-	case common.SensorComponentEventCentralReachable:
-		m.resetContext()
-		m.resetLastSentState()
-		m.centralReady.Signal()
-		m.enricherTicker.Reset(tickerTime)
-	case common.SensorComponentEventOfflineMode:
-		m.centralReady.Reset()
-		m.enricherTicker.Stop()
-	}
-}
+func (m *networkFlowManager) Notify(_ common.SensorComponentEvent) {}
```

Comment:
I wouldn't delete this code. I think it would be good to be able to fallback to v1 if the FF for v3 is not enabled.

Reply:
Done